### PR TITLE
Remove deprecated GraphViz (.dot file) support

### PR DESCRIPTION
**Pull request by tbennun**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -165,6 +164,7 @@ perfdata.db
 
 .idea/
 _dotgraphs/
```
Comment (definelicht) [inline .gitignore:166]: Do you wanna remove this?

File: dace/config_schema.yml [hunk 2]
```diff
@@ -38,23 +38,17 @@ required:
...
-                description: Open a GraphViz window after every transformation.
-
             visualize_sdfv:
                 type: bool
                 default: false
                 title: Visualize SDFG
                 description: Open a SDFV in browser every transformation.
 
-            savedots:
+            saveopts:
```
Comment (definelicht) [inline dace/config_schema.yml:47]: Title is not so descriptive, maybe "save_intermediate"?

File: .gitignore (hunk 1 above)
Comment (tbennun) [reply .gitignore:166]: backwards compatibility with existing git repos :P

File: dace/config_schema.yml (hunk 2 above)
Comment (tbennun) [reply dace/config_schema.yml:47]: I was thinking about it. Changing.

**Pull request merged**